microcosm_flask/conventions/encoding.py

```diff
@@ -170,3 +191,36 @@ def require_response_data(response_data):
     if not response_data:
         raise NotFound
     return response_data
+
+
+def csvify(response_data, response_schema):
```

Comment:
It would be wise to organize csv code into another module, especially if you break up formatting into a base/json/csv scheme.